js/repl/PluginConfig.js

```diff
@@ -1,11 +1,62 @@
 // @flow
+import camelCase from "lodash.camelcase";
+import type {
+  PluginConfig,
+  MultiPackagesConfig,
+  ReplState,
+  EnvFeatures,
+} from "./types";
 
-import type { PluginConfig, ReplState } from "./types";
+const normalizePluginName = pluginName =>
+  `_babel_${camelCase(`plugin-${pluginName}`)}`;
+
+const babelConfig: PluginConfig = {
+  label: "Babel",
+  package: "babel-standalone",
+  version: "6",
+  baseUrl: "https://unpkg.com",
+  instanceName: "Babel",
+};
 
 const envPresetConfig: PluginConfig = {
   label: "Env Preset",
   package: "babel-preset-env-standalone",
-  version: "0",
+  version: "1.6.2",
+  baseUrl: "https://unpkg.com",
+  versionKey: "envVersion",
+  instanceName: "babelPresetEnv",
+};
+
+const stage3Plugins: Array<PluginConfig> = [
+  "proposal-async-generator-functions",
+  // "proposal-object-rest-spread",
```

Comment:
oops, I probably confused you, but I meant to say that we should add a comment describing _why_ we need `stage3Plugins` (i.e. proposal-async-generator-functions not available >= beta.36)